net.krafting.SemantiK.Lang.English.yml

```diff
@@ -0,0 +1,25 @@
+app-id: net.krafting.SemantiK.Lang.English
+build-extension: true
+separate-locales: false
+branch: '1.0.0'
+runtime: net.krafting.SemantiK
+runtime-version: stable
+sdk: org.freedesktop.Sdk//23.08
+
+modules:
+  - name: EnglishLang
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 english.list /app/share/extra/English/words.list
+      - gzip -d GoogleNews-vectors-negative300-SLIM.bin.gz
+      - install -Dm644 GoogleNews-vectors-negative300-SLIM.bin /app/share/extra/English/model.bin
+      - install -Dm644 net.krafting.SemantiK.Lang.English.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo
+    sources:
+      - type: file
+        path: english.list
+      - type: file
+        path: net.krafting.SemantiK.Lang.English.metainfo.xml
+      - type: file
+        url: https://github.com/eyaler/word2vec-slim/raw/master/GoogleNews-vectors-negative300-SLIM.bin.gz
```

Comment:
you should NEVER pull from a branch. You should pull a specific tag or commit.

Reply:
Fixed, used a commit and a separate step to download it.